projects/core/src/error-handling/error-interceptors/logger-error.interceptor.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, inject } from '@angular/core';
+import { LoggerService } from '../../logger';
+import {
+  ChainedErrorInterceptorFn,
+  ErrorInterceptor,
+  ErrorInterceptorPriority,
+} from './error-interceptor';
+
+Injectable({
```

Comment:
[MEDIUM]
Missing `@`. Does it compile?
```suggestion
@Injectable({
```

Reply:
yes, without any errors! that's strange...

Reply:
If it didn't throw any errors, was this class even used in runtime? Did you check if our app is still logging stuff to the console in case of errors?